packages/babel-preset-gatsby/src/index.js

```diff
@@ -45,6 +45,18 @@ module.exports = function preset(_, options = {}) {
     }
   }
 
+  const plugins = []
```

Comment:
We shouldn't need this now that you've added it to the array in the bottom right?

Reply:
Oh, that's my bad - I didn't remove that part before merging - I'll fix this up